tuf-spec.md

```diff
@@ -833,12 +833,12 @@ repo](https://github.com/theupdateframework/specification/issues).
    cryptographic hash function.  For example: { "sha256": HASH, ... }. HASH is
    the hexdigest of the cryptographic function computed on the target file.
 
-   If defined, the elements and values of "custom" will be made available to the
-   client application.  The information in "custom" is opaque to the framework
-   and can include version numbers, dependencies, requirements, and any other
-   data that the application wants to include to describe the file at
-   TARGETPATH.  The application may use this information to guide download
-   decisions.
+   If defined, the elements and values of the CUSTOM object will be made
+   available to the client application.  The format of, and information in,
```

Comment:
Minor nit: Do you think that it is clear that the CUSTOM object, if supplied, must be a json object (i.e. deserialize to a Python dictionary), if we say the "format ... is opaque"?

Reply:
That's a good question. I wanted to try and avoid mentioning JSON in the description, because we are trying to make the actual format of the metadata looser in the specification.

I felt like the change here was as clear, from the prose alone, as other places we talk about objects. Though admittedly those other places in the specification where we talk about objects also have an explicit example of the object format and don't refer to it as opaque.

I will try and clarify this some more tomorrow. Either by clarifying the language here, by pointing out in section 4.1 that all objects in the file formats are JSON objects, or perhaps a combination.

Reply:
I think it is not a very big deal. I guess my association with the word "format" here is biased through `securesystemslib.formats`, where the format also specifies the datatype. It's quite likely that others don't have that association.

Reply:
I've pushed a fixup commit with some further clarification that the framework only needs to know that the value mapped to the "custom" attribute is an object.